### PR TITLE
PACA-1020: SLO adjustments

### PR DESCRIPTION
**Pull request by KyRobbins**
**JIRA Ticket:**
[PACA-1020](https://jira.cms.gov/browse/PACA-1020)

**User Story or Bug Summary:**

---

### What Does This PR Do?

To prepare for the full launch of the PACA resources, we are making small adjustments to the SLO requirements for PACA endpoints.
 - Remove per_kb SLOs as they have no obvious value currently, and have a high chance to set off false alarms
 - Divert PACA SLO alarms to the `bfd-test` slack channel during initial full launch period till SLOs can be refined.

### What Should Reviewers Watch For?

If you're reviewing this PR, please check for these things in particular:

* Verify all PR security questions and checklists have been completed and addressed.

### What Security Implications Does This PR Have?

Submitters should complete the following questionnaire:

* If the answer to any of the questions below is **Yes**, then **you must** supply a link to the associated Security Impact Assessment (SIA), security checklist, or other similar document in Confluence here: **N/A**

    * Does this PR add any new software dependencies? 
      * [ ] Yes
      * [x] No
    * Does this PR modify or invalidate any of our security controls?
      * [ ] Yes
      * [x] No
    * Does this PR store or transmit data that was not stored or transmitted before?
      * [ ] Yes
      * [x] No

* If the answer to any of the questions below is **Yes**, then please add @StewGoin as a reviewer, and note that this PR **should not be merged** unless/until he also approves it.
    * Do you think this PR requires additional review of its security implications for other reasons?
      * [ ] Yes
      * [x] No

### What Needs to Be Merged and Deployed Before this PR?

This PR cannot be either merged or deployed until the following prerequisite changes have been fully deployed:

* N/A

### Submitter Checklist

I have gone through and verified that...:

* [x] I have named this PR and branch so they are [automatically linked](https://confluence.atlassian.com/adminjiracloud/integrating-with-development-tools-776636216.html) to the (most) relevant Jira issue. Ie: `BFD-123: Adds foo`
* [x] This PR is reasonably limited in scope, to help ensure that:
    1. It doesn't unnecessarily tie a bunch of disparate features, fixes, refactorings, etc. together.
    2. There isn't too much of a burden on reviewers.
    3. Any problems it causes have a small "blast radius".
    4. It'll be easier to rollback if that becomes necessary.
* [x] This PR includes any required documentation changes, including `README` updates and changelog / release notes entries.
* [x] The data dictionary has been updated with any field mapping changes, if any were made.
* [x] All new and modified code is appropriately commented, such that the what and why of its design would be reasonably clear to engineers, preferably ones unfamiliar with the project.
* [x] All tech debt and/or shortcomings introduced by this PR are detailed in `TODO` and/or `FIXME` comments, which include a JIRA ticket ID for any items that require urgent attention.
* [x] Reviews are requested from both:
    * At least two other engineers on this project, at least one of whom is a senior engineer or owns the relevant component(s) here.
    * Any relevant engineers on other projects (e.g. DC GEO, BB2, etc.).
* [x] Any deviations from the other policies in the [DASG Engineering Standards](https://github.com/CMSgov/cms-oeda-dasg/blob/master/policies/engineering_standards.md) are specifically called out in this PR, above.
    * Please review the standards every few months to ensure you're familiar with them.

### CODE REVIEW

**Review by daryll — APPROVED** [review]

**Review by brianburton — DISMISSED** [review]
LGTM

File: ops/terraform/modules/resources/bfd_server_slo_alarms/main.tf [hunk 1]
```diff
@@ -908,8 +908,8 @@ resource "aws_cloudwatch_metric_alarm" "slo_claim_no_resources_latency_mean_15m_
   metric_name = local.metrics.claim_no_resources_latency
   namespace   = local.namespace
 
-  alarm_actions = local.alert_arn
-  ok_actions    = local.alert_ok_arn
+  alarm_actions = data.aws_sns_topic.bfd_test_sns.arn
```
Comment (mjburling) [inline ops/terraform/modules/resources/bfd_server_slo_alarms/main.tf:911]:
```suggestion
  alarm_actions = [data.aws_sns_topic.bfd_test_sns.arn]
```

File: ops/terraform/modules/resources/bfd_server_slo_alarms/main.tf [hunk 2]
```diff
@@ -932,56 +932,8 @@ resource "aws_cloudwatch_metric_alarm" "slo_claim_no_resources_latency_mean_15m_
...
-    "exceeded WARNING SLO threshold of 600 ms/KB for ${local.app} in ${var.env} environment.",
-    "\n\n${local.dashboard_message_fragment}"
-  ])
-
-  metric_name = local.metrics.claim_resources_latency_by_kb
-  namespace   = local.namespace
-
-  alarm_actions = local.warning_arn
-  ok_actions    = local.warning_ok_arn
+  alarm_actions = data.aws_sns_topic.bfd_test_sns.arn
```
Comment (mjburling) [inline ops/terraform/modules/resources/bfd_server_slo_alarms/main.tf:935]:
```suggestion
  alarm_actions = [data.aws_sns_topic.bfd_test_sns.arn]
```

File: ops/terraform/modules/resources/bfd_server_slo_alarms/main.tf [hunk 3]
```diff
@@ -1014,42 +966,8 @@ resource "aws_cloudwatch_metric_alarm" "slo_claim_with_resources_bulk_latency_99
...
-
-  metric_name = local.metrics.claim_resources_latency_by_kb
-  namespace   = local.namespace
-  dimensions = {
-    client_ssl = data.external.client_ssls_by_partner["claim_resources_latency_by_kb"].result[each.key]
-  }
-
-  alarm_actions = local.warning_arn
-  ok_actions    = local.warning_ok_arn
+  alarm_actions = data.aws_sns_topic.bfd_test_sns.arn
```
Comment (mjburling) [inline ops/terraform/modules/resources/bfd_server_slo_alarms/main.tf:969]:
```suggestion
  alarm_actions = [data.aws_sns_topic.bfd_test_sns.arn]
```

File: ops/terraform/modules/resources/bfd_server_slo_alarms/main.tf [hunk 4]
```diff
@@ -1072,8 +990,8 @@ resource "aws_cloudwatch_metric_alarm" "slo_claimresponse_no_resources_latency_m
   metric_name = local.metrics.claimresponse_no_resources_latency
   namespace   = local.namespace
 
-  alarm_actions = local.alert_arn
-  ok_actions    = local.alert_ok_arn
+  alarm_actions = data.aws_sns_topic.bfd_test_sns.arn
```
Comment (mjburling) [inline ops/terraform/modules/resources/bfd_server_slo_alarms/main.tf:993]:
```suggestion
  alarm_actions = [data.aws_sns_topic.bfd_test_sns.arn]
```

File: ops/terraform/modules/resources/bfd_server_slo_alarms/main.tf [hunk 5]
```diff
@@ -1096,56 +1014,8 @@ resource "aws_cloudwatch_metric_alarm" "slo_claimresponse_no_resources_latency_m
...
-    "exceeded WARNING SLO threshold of 1000 ms/KB for ${local.app} in ${var.env} environment.",
-    "\n\n${local.dashboard_message_fragment}"
-  ])
-
-  metric_name = local.metrics.claimresponse_resources_latency_by_kb
-  namespace   = local.namespace
-
-  alarm_actions = local.warning_arn
-  ok_actions    = local.warning_ok_arn
+  alarm_actions = data.aws_sns_topic.bfd_test_sns.arn
```
Comment (mjburling) [inline ops/terraform/modules/resources/bfd_server_slo_alarms/main.tf:1017]:
```suggestion
  alarm_actions = [data.aws_sns_topic.bfd_test_sns.arn]
```

File: ops/terraform/modules/resources/bfd_server_slo_alarms/main.tf [hunk 6]
```diff
@@ -1178,42 +1048,8 @@ resource "aws_cloudwatch_metric_alarm" "slo_claimresponse_with_resources_bulk_la
...
-
-  metric_name = local.metrics.claimresponse_resources_latency_by_kb
-  namespace   = local.namespace
-  dimensions = {
-    client_ssl = data.external.client_ssls_by_partner["claimresponse_resources_latency_by_kb"].result[each.key]
-  }
-
-  alarm_actions = local.warning_arn
-  ok_actions    = local.warning_ok_arn
+  alarm_actions = data.aws_sns_topic.bfd_test_sns.arn
```
Comment (mjburling) [inline ops/terraform/modules/resources/bfd_server_slo_alarms/main.tf:1051]:
```suggestion
  alarm_actions = [data.aws_sns_topic.bfd_test_sns.arn]
```

**Review by mjburling — CHANGES_REQUESTED** [review]
Just as we discussed, this looks great! 👍 

Update those `alarm_actions` values to be collections, and I think we'll be set.

**Review by mjburling — APPROVED** [review]

**Review by malessi — APPROVED** [review]
After updating all `alarm_actions` to collections, everything looks good to me!

**Review by daryll — APPROVED** [review]

**Pull request merged**